aws/resource_aws_ec2_client_vpn_network_association_test.go

```diff
@@ -0,0 +1,154 @@
+package aws
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/service/ec2"
+	"github.com/hashicorp/terraform/helper/acctest"
+	"github.com/hashicorp/terraform/helper/resource"
+	"github.com/hashicorp/terraform/terraform"
+)
+
+func TestAccAwsEc2ClientVpnNetworkAssociation_basic(t *testing.T) {
+	var assoc1 ec2.TargetNetwork
+	rStr := acctest.RandString(5)
+
+	resource.Test(t, resource.TestCase{
```

Comment:
This acceptance testing should use `resource.ParallelTest()` instead of `resource.Test()`

```suggestion
	resource.ParallelTest(t, resource.TestCase{
```